mirrormanager2/app.py

```diff
@@ -380,7 +383,13 @@ def host_new(site_id):
             APP.logger.debug('Could not create the new host')
             APP.logger.exception(err)
 
+        message = dict(
+            status=status,
```

Comment:
Hm, after thinking about it a bit it seems like the answer here is probably to not send the message.

@Devyani-Divs, could you move the publish line up inside the ``try/except`` block above just after the ``flask.flash`` line?  That way, if the ``SESSION.flush()`` line produces some sort of error, then we won't publish a message that says "a host was added" when the host was not actually added due to some error.